randovania/resolver/bootstrap.py

```diff
@@ -55,6 +79,14 @@
         # Light Ammo
         46: 100
     },
+    RandovaniaGame.METROID_PRIME_CORRUPTION: {
+        # Missile
+        4: 100,
+        # Energy Tank
+        20: 14,
+        # Ship Missile
+        45, 11
```

Comment:
`45: 11`

Reply:
Would it make sense to add Energy Cells into the list here?